imports.py

```diff
@@ -12,193 +10,283 @@
+                print("resourceType:", response_dict['entry'][0]['resource']['resourceType'],
+                 end="  ")
+                print("id: " + response_dict['entry'][0]['resource']['id'])
+            else:
+                print("Not Present")
+            print("") 
+               
+    def put_patient(self, file):
+        org_file = open(file, "r")
+        form_data = org_file.read()
+        response = self.tims_client.do_hapi_post_request("/Patient/", 
+                       form_data,  {"_format" : "json", "_pretty" : "true"})    
+        
+    def get_patient(self, name):
+        response = self.tims_client.do_hapi_get_request("/Patient/", 
+                       {'name' : name})
+        self.decode_patient_json(response)  
+       
+    def decode_patient_json(self, response_dict):
+            if ('entry' in response_dict):
+                print("name:" + response_dict['entry'][0]['resource']['name'][0]['family'])
+                print("name:" + response_dict['entry'][0]['resource']['name'][0]['given'][0])            
+                print("resourceType:", response_dict['entry'][0]['resource']['resourceType'],
+                 end="  ")
+                print("id: " + response_dict['entry'][0]['resource']['id'])
+            else:
+                print("Not Present")
+            print("")
+            
+    def put_encounter(self, file):
+        org_file = open(file, "r")
+        form_data = org_file.read()
+        response = self.tims_client.do_hapi_post_request("/Encounter/", 
+                       form_data,  {"_format" : "json", "_pretty" : "true"})    
+        
+    def get_encounter(self, name):
+        response = self.tims_client.do_hapi_get_request("/Encounter/", 
+                       {'patient' : name})
+        self.decode_encounter_json(response)  
+            
+    def decode_encounter_json(self, response_dict):
+            if ('entry' in response_dict):
+                print(response_dict['entry'][0]['fullUrl'])   
+                print("id: " + response_dict['entry'][0]['resource']['id'])
+                print("text status: " + response_dict['entry'][0]['resource']['text']['status'])
+                print("text div: " + response_dict['entry'][0]['resource']['text']['div'])            
+                print("status: " + response_dict['entry'][0]['resource']['status'])
+            else:
+                print("Not Present")
+            print("")        
+
+        
 # Set true and run this file directly for troubleshooting / ease of development
 DEBUG = False
 if __name__ == '__main__':
-	if DEBUG:
-		client = TimsClient(base_url=BASE_URL, use_cache=True)
-		# examples:
-		client.summarize_code_system_by_name('LOINC')
-		client.lookup_code(system='http://loinc.org', code='LA6668-3')
+    if DEBUG:
+        BASE_URL = "http://20.119.216.32:8000/r4/"  # TIMS server for 
+        client = TimsClient(base_url=BASE_URL, use_cache=True)
```

Comment:
I though the BASE_URL here was a little confusing with the one passed in, so I scooted it down here to be closer to the scope where it's used.